packages/cyberstorm/src/components/Button.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { MouseEventHandler } from "react";
+import styles from "./componentStyles/Button.module.css";
+
+export interface ButtonProps {
```

Comment:
All the props are optional, so the button generated by passing no props is a bit of sadge. Should we show some default text if label nor icons are passed as props? Just a thought, not necessarily required as a part of this PR.

Reply:
I don't like default content. The button should in theory always have some content, if it is used in a sensible way. Kind of adds unnecessary complexity imo.

Reply:
Yes, **if** it's used in a sensible way 😄 . Anyways, fine by me, it can always be changed later if we run into problems.